backend/siarnaq/bracket/challonge.py

```diff
@@ -141,17 +141,63 @@ def get_round_indexes(tournament: Tournament, *, is_private: bool):
     return round_indexes
 
 
-def get_match_and_participant_objects_for_round(
-    round: TournamentRound, *, is_private: bool
-):
-    tournament_data = get_tournament_data(round.tournament, is_private=is_private)
-    # Derive match dicts/JSON objects (that Challonge gives us) of this round
+def _pair_public_private_challonge_ids(tournament: Tournament):
```

Comment:
I feel like this function does the same thing twice: once on items with type `match` and once with `participant`… and the behaviour is identical for both cases.

Perhaps accept the type as a function argument?